src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -15,40 +15,31 @@ tags:
 
 <Intro>
 
-When you create an app, you'll likely want to share it. From New Relic One's **Apps** page, you can subscribe to apps you create, publish, and deploy, and to other publicly available apps.
-
-You must have the **Nerdpack manager** role to subcribe accounts to apps. Read on to learn about permissions and versions.
+Details about permissions required for creating and managing [New Relic One applications](https://developer.newrelic.com/build-apps).
 
 </Intro>
 
-## Permissions for managing applications
-
-The Nerdpack manager role is a [New Relic add-on role](https://docs.newrelic.com/docs/accounts/accounts/roles-permissions/add-roles-permissions). When you create a Nerdpack, you have the Nerdpack manager role for handling that Nerdpack. New Relic account administrators have the Nerdpack manager role automatically, and can subscribe their accounts to available Nerdpacks.
-
-User permissions vary depending on which pricing plan you are on.
-
-### New Relic One pricing plan
-
-For accounts with New Relic One pricing, there are permissions differences for basic users and full users:
+There are several factors affecting your New Relic One app permissions. 
 
-**Full users** have the Nerdpack manager role and have full capabilities for creating and managing New Relic One applications, as well as accessing all types of applications in the New Relic One catalog.
+## Basic user restrictions 
 
-A **basic user** can develop and view their own local New Relic One apps, but they ***cannot***:
+The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type): 
 
-- Subscribe other users to apps they’ve created.
-- Access or manage apps in the New Relic One catalog.
-- Access apps in the entity explorer sidebar.
+* **Basic users** can only develop New Relic One apps locally. They can’t publish or subscribe to an app, whether it's deployed in their account, available in [the public catalog](https://opensource.newrelic.com/nerdpacks/), or one they've built. 
+* **Full users** can theoretically do everything, but there may be restrictions due to custom role assignments (see below). 
 
-### Original product-based pricing
+## Role-related permissions 
 
-For accounts on our original product-based pricing, here are access details:
+Full users have role-related rules that impact their ability to assign New Relic One apps and subscribe accounts to them. How this works depends on your account/user model: 
```

Comment:
```suggestion
Full users have role based permissions that impact their ability to assign New Relic One apps and subscribe accounts to them. How this works depends on your account/user model: 
```

Reply:
Also what' does "assign" me in this context?

Reply:
i made an edit based on this. Basically was trying to communicate that there are limits on being able to subscribe accounts to NR1 apps.